### PR TITLE
allow late migration

### PR DESCRIPTION
**Pull request by msfeldstein**
Proposal to allow tokens to be migrated after the lockup period, in order to let us decouple the lockup period from 100% of token holders migrating.

### CODE REVIEW

File: contracts/PartyToken.sol [hunk 1]
```diff
@@ -5,18 +5,25 @@ pragma solidity 0.8.5;
...
 
 /*
 Party Token
 by Anna Carroll
 */
-contract PartyToken is ERC20VotesComp, Pausable {
+contract PartyToken is ERC20VotesComp {
+    // ============ Events ==============
+    
+    event Unpaused(address account);
```
Comment (anna-carroll) [inline contracts/PartyToken.sol:16]: don't need to emit the address of the account that un-pauses because we know it has to be PartyDAO multisig!

**Comment by msfeldstein** [comment]
yeah i wasn't sure if we needed that event at all but i wanted to match the tests

File: contracts/PartyToken.sol [hunk 2]
```diff
@@ -5,18 +5,25 @@ pragma solidity 0.8.5;
...
+
     // ============ Immutables ============
 
     address public immutable partyDAOMultisig;
     address public immutable deprecationContract;
 
+    // ======== State =========
+
+    bool lockupPeriod = true;
```
Comment (anna-carroll) [inline contracts/PartyToken.sol:25]: bikeshedding, but i would call it `lockupIsActive`, as `lockupPeriod` sounds more like i'll get a uint256 number of blocks for which lockup is active

File: contracts/PartyToken.sol (hunk 1 above)
Comment (anna-carroll) [reply contracts/PartyToken.sol:16]: I would also rename to `LockupEnded` or `Unlocked` or something

File: contracts/PartyToken.sol [hunk 3]
```diff
@@ -25,15 +32,15 @@ contract PartyToken is ERC20VotesComp, Pausable {
         deprecationContract = _deprecationContract;
         // mint 10M totalSupply to partyDAO multisig
         _mint(_partyDAOMultisig, 10_000_000 * (10 ** 18));
-        // pause transfers; unpaused when lockup ends
-        _pause();
     }
 
     // ======== External Functions =========
 
     function endLockup() external {
```
Comment (anna-carroll) [inline contracts/PartyToken.sol:39]: we could rename this to `unlock()` or leave as-is

**Pull request closed without merging**